app/views/school-administrator/dashboard/SchoolAdminDashboardTeacherView.vue

```diff
@@ -7,6 +7,7 @@
 <template>
     <loading-progress :loading-status="loadingStatuses">
         <div v-if="!loading">
+            <breadcrumbs v-if="!loading" :links="links"></breadcrumbs>
```

Comment:
I think the `v-if` in the div above should cover this `v-if`?  Could be wrong though - does this fail without it?